packages/core/src/geometry/Buffer.js

```diff
@@ -11,8 +13,11 @@ export default class Buffer
 {
     /**
      * @param {ArrayBuffer| SharedArrayBuffer|ArrayBufferView} data the data to store in the buffer.
+     * @param {boolean} [_static=true] `true` for static buffer
+     * @param {boolean} [index=false] `true` for index buffer
+     * @param {boolean} [shared=false] if `true`, buffer will stay alive when geometry is disposed
      */
-    constructor(data, _static = true, index = false)
```

Comment:
Hello! There must be a way of not having to pass in shared as a property.
Could we do object counting instead?

Reply:
Wouldn't that take a serious effort to add?

Reply:
I don't want to open this can of worms.

We can remove it from constructor, its fine as a property. If user makes shared buffer, he needs extra operations with it anyway and one more line is not a problem

Reply:
@Dairnon agreed, It definitely would take more effort. I see a buffer the same way as I see a texture. You would not create a texture and declare it to be shareable iyswim?